README.md

```diff
@@ -1,11 +1,12 @@
 # CircleCI Concurrency Control Orb
 
+[![CircleCI](https://circleci.com/gh/eddiewebb/circleci-queue/tree/master.svg?style=svg)](https://circleci.com/gh/eddiewebb/circleci-queue/tree/master) [![GitHub license](https://img.shields.io/github/license/eddiewebb/circleci-queue)](https://github.com/eddiewebb/circleci-queue/blob/master/LICENSE)
```

Comment:
License is fine.   Markdown will allow the images on their own lines, but show inline. This presents the same, but much easier to read source.

https://jbt.github.io/markdown-editor/#pY4xC8IwEIX3/oqKix3M7YI41KIF6eJYHEx6JAdJo83F4r9Xa1GXTm7Hu/c+vnlaYZ9uvYoOW06Selbn1CmLeXlaGOZLWAGoIVEklHegDWDTEPYo5eezvEaMCNwhgjsHxk6Em94EvltcP6/sP1j28toR76NMLSlsA36B5LQIhtA2QZAHTWyihLE2if8xei8Gn0kZab0cZeBQ5kV1LLIH

Reply:
Done with changes